pkg/provision/buildroot.go

```diff
@@ -92,6 +92,13 @@ func (p *BuildrootProvisioner) GenerateDockerOptions(dockerPort int) (*provision
 	driverNameLabel := fmt.Sprintf("provider=%s", p.Driver.DriverName())
 	p.EngineOptions.Labels = append(p.EngineOptions.Labels, driverNameLabel)
 
+	noPivot := true
+	// Using pivot_root is not supported on fstype rootfs
+	if fstype, err := rootFileSystemType(p); err == nil {
+		log.Debugf("root file system type: %s", fstype)
+		noPivot = fstype == "rootfs"
+	}
```

Comment:
I would suggest making this a warning -- again, `--no-pivot` is fundamentally insecure.

Reply:
Note that switching to stat broke everything, since it has a different return value

Reply:
For some reason, stat thinks that the file system is called "ramfs" (not "rootfs")

Reply:
Added a warning, not likely to be seen - but only valid for old ISO, not necessary for new ISO

Reply:
`ramfs` === `rootfs` (both tools are fetching the filesystem using the same method -- `statfs(2)` and checking the filesystem magic number against `RAMFS_SUPER_MAGIC`). But I don't have a strong opinion either way. Thanks.